### ISSUE TITLE
a11y_SchemaRenderer> Modal> 1 violations is appearing

### ISSUE THREAD

**Issue by priya-rajput** [cengage/react-magma#527]
**Describe the bug**
SchemaRenderer>> "1 violations" is appearing for "Modal". 

**To Reproduce**
1. Go to 'https://storybook-preview-290--upbeat-sinoussi-f675aa.netlify.app/?path=/story/schemarenderer--modal'
2. Click on 'Accessibility" add on
3. "1 violations" is appearing

**Expected behavior**
"0 violations" should appear for SchemaRenderer> Modal.

**Screenshots**
https://somup.com/c3n26dTcho

**Desktop :**
 - OS: [Win 10]
 - Browser [chrome]
 - Version [Version 98.0.4758.82 (Official Build) (64-bit)]
